recipe/meta.yaml

```diff
@@ -22,7 +24,7 @@ requirements:
     - pip
     - setuptools
   run:
-    - python >=3.5
+    - python >=3.5|2.7.*
```

Comment:
This supports py2k? If so you can remove these restrictions outright (also on the one in host)

Reply:
Yes, PY2 support is back.  Restrictions removed.